encoder_field.go

```diff
@@ -153,10 +153,16 @@ func typeFields(t reflect.Type) []field {
 				if sf.PkgPath != "" && !sf.Anonymous { // unexported
 					continue
 				}
-				tag := sf.Tag.Get("json")
+
+				tag := sf.Tag.Get("velocypack")
+				if len(tag) == 0 {
```

Comment:
for the time being len(tag) is always 0 but in go-driver I used that on the branch with Merkle tree:
type RevisionTree struct {
	Version  int                `json:"version"`
	RangeMin RevisionInt64      `json:"rangeMin,string" velocypack:"string,noquota"`
	RangeMax RevisionInt64      `json:"rangeMax,string" velocypack:"string,noquota"`
	Nodes    []RevisionTreeNode `json:"nodes"`
}